src/server/utils/lyft-helper.js

```diff
@@ -1,196 +1,285 @@
-    headers: headers,
-    body: JSON.stringify(body)
-  })
-  .then(function (res) {
-    return res.json();
-  })
-  .then(function (data) {
-    console.log('successful phoneNumber post LYFT', data);
-  })
-  .catch(function (err) {
-    console.log('error post of phoneNumber LYFT', err);
-  });
+      method: 'POST',
+      headers: headers,
+      body: JSON.stringify(body)
+    })
+    .then(res => {
+      return res.json();
+    })
+    .then(data => {
+      console.log('successful phoneNumber post LYFT', data);
+    })
+    .catch(err => {
+      console.log('error post of phoneNumber LYFT', err);
+    });
 };
 
-var lyftPhoneCodeAuth = function (fourDigitCode, phoneNumber, userLocation, userId) {
+export const lyftPhoneCodeAuth = (req, res) => {
+  let fourDigitCode = req.body.fourDigitCode;
+  let phoneNumber = req.body.phoneNumber;
+  let userLocation = req.body.userLocation || null;
+  let userId = req.body.userId || null; // alexaUserId
 
-  userId = userId || null; // alexaUserId
-  userLocation = userLocation || null;
-
-  var url = lyftMethods.phoneCodeAuth.path;
-  var headers = lyftMethods.phoneCodeAuth.headers();
-  var body = lyftMethods.phoneCodeAuth.body(fourDigitCode, phoneNumber, userLocation);
+  let url = phoneCodeAuth.path;
+  let headers = phoneCodeAuth.headers();
+  let body = phoneCodeAuth.body(fourDigitCode, phoneNumber, userLocation);
 
   console.log('lyftPhoneCodeAuth number', fourDigitCode, phoneNumber, 'body', body, 'headers', headers, 'url', url);
 
   fetch(url, {
-    method: 'POST',
-    headers: headers,
-    body: JSON.stringify(body)
-  })
-  .then(function (res) {
-    return res.json();
-  })
-  .then(function (data) {
-    // console.log('successful phoneCodeAuth post LYFT', data);
-
-    // the responseMethod function returns an object with the parameters we need for subsequent operations only, and in a key-name generalised manner.
-    var response = lyftMethods.phoneCodeAuth.responseMethod(data, userId);
-
-    // POST THE USER DATA TO OUR RELATIONAL DATABASE
-    var dbpostURL = 'http://' + APIserver + '/users/updateOrCreate';
-
-    fetch(dbpostURL, {
       method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-        'x-access-token': APItoken
-      },
-      body: JSON.stringify(response)
+      headers: headers,
+      body: JSON.stringify(body)
     })
-    .then(function (res) {
+    .then(res => {
       return res.json();
     })
-    .then(function (data) {
-      console.log('success posting user', data);
+    .then(data => {
+      // the responseMethod function returns an object with the parameters we need for subsequent operations only, and in a key-name generalised manner.
+      let response = phoneCodeAuth.responseMethod(data, userId);
+
+      // POST THE USER DATA TO OUR RELATIONAL DATABASE
+      // let dbpostURL = 'http://' + APIserver + '/users/updateOrCreate';
+      let dbpostURL = 'http://localhost:8080/users/updateOrCreate';
```

Comment:
can we please get rid of this localhost reference and instead assign APIserver at the top of the file based on env? its actually already done like that on master right now, so this seems like a regression / omission of merging that code from master